source/support.tex

```diff
@@ -444,18 +498,43 @@
 \end{codeblock}
 
 \pnum
+For all members declared
+\tcode{static} \tcode{constexpr}
+in the
+\tcode{numeric_limits}
+template, specializations shall define these values in such a way
+that they are usable as
+constant expressions.
```

Comment:
Ah, this applies to user specializations. Seems weird to put a "shall" on that, but OK.